### DISCUSSION TITLE
Control type of search keyword

### DISCUSSION THREAD

**Discussion by vuon9** [miguelmota/cointop#253] [Ideas]
Hi, I suggest having these kinds of search keywords to make it easier and controllable.

`/s:btc` -> search by symbol only
`/n:bitcoin` -> search by name only
`/bitcoin` -> default behavior (by symbol, then by name)

Here is a draft PR:
https://github.com/cointop-sh/cointop/pull/254

**Comment by lyricnz** [comment]
As mentioned in the PR:

The search behaviour changed recently - now it searches from where the cursor currently is, so you can do repeated search (by hitting / [enter]).

Yes, there is kindof an edge case where if you search for something slightly obscure (say "coin") it first goes to name/symbol=coin, then each repeat starts from that point (with a substring/partial search). It doesn't seem to be possible to do an approximate search before the full-patch result.
